resources/constraints/constraints_py3.11.txt

```diff
@@ -0,0 +1,536 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
it should be done using python 3.11?

Reply:
Yeah, sorry I could not get pip-compile to spit out something on Python 3.11 it hangs on my machine so, I copied the 3.10 file.

Reply:
I have updated part of this PR connected with python 3.11 constraints.